### ISSUE TITLE
Change "Enroll" page text for new application period

### ISSUE THREAD

**Issue by smarziano** [smartchicago/chicago-early-learning#770]
# Apply Today
_If you are interested in programs for children birth to aged three – contact your preferred provider directly._

## Are you ready to apply for preschool for your 3 or 4 year old?
Enrolling your 3 or 4 year-old child in an early learning program is easy, and open year-round. Follow these easy steps to get your child enrolled in preschool:
 **Step 1)**[ Click Here](https://cys.mycopa.com/familyPortal/welcome.epl) to create an online account for your family

**Step 2)** Fill in information on your family, such as employment and income, information on your child, and then apply for your top choice preschool site

**Step 3)** If your child qualifies, you will receive an email on June 1st with information on how to verify your child’s preschool seat

See you for the first day of school in September!

If you have questions, please call our Chicago Early Learning Hotline: 312-229-1690.

## Have you found the right program for your child?
The City of Chicago and its partners offer early learning programs at over 600 sites throughout the city. Visit [our search page](https://chicagoearlylearning.org/search/) to identify sites in areas near your home or work. The Chicago Early Learning Portal allows you to identify a program’s quality rating, schedule and ages served.

[Add screenshot of compare & apply tool that is on the homepage]

We encourage you to reach out to preferred providers to ask questions or schedule a visit. Resources to help identify if a program is a good fit for your family are included on [our family information page.](https://chicagoearlylearning.org/families)

## Need application support?
If you have any concerns with the online application, please call our Family Support Hotline at 312-229-1690, or visit any of the Family Resource Centers listed below.

You are welcome to visit a Chicago Public School or community-based preschool site directly, within their operating hours, for additional support.

## How will you verify your child's placement?
If your child is offered a preliminary preschool placement, you will receive an email with verification instructions.

Follow these three steps to accept your child’s placement:
### Step 1) Gather Documents 
***see below for a list of document types that are generally accepted**
Proof of your child’s birth –  Examples: Birth certificate, passport or state issued medical card
Proof of employment or other income for all family members listed on the application – Examples: W-2, two pay-stubs 

### Step 2) Bring and verify Documents
If your child has a seat at a **Chicago Public School (CPS) site**

[insert table - #768]

If your child has a seat at a **Community-based site** bring your documents directly to the site which your child was assigned.
 
Bring your documents to the Community-Based Site at which your child was assigned

### Proof of child’s age
- Original birth certificate
- Passport

### Proof of address
- Driver’s license/ID card
- Utility bill
- Lease agreement/mortgage statement
- Current voter’s registration card

### Proof of income
- Current paystub
- 1040/W-2
- SSI Letter
- Unemployment letter/stub
- Workers’ compensation letter/stub
- Retirement or pension stub/letter
- Alimony
- Official child support letter
- TANF
- Income verification form

_Note:  If your family is in a temporary living situation, you are not required to present the documents listed above for verification.  All schools must immediately enroll students in temporary living situations even if the student lacks health, immunization or school records, proof of guardianship, proof of residency, or any other documentation normally required for school enrollment._

**Comment by smarziano** [comment]
Apply Today: ¡Aplique hoy!
_Si usted está interesado(a) de un programa para niños recién nacidos hasta la edad de tres años, comuníquese con su proveedor directamente._

## ¿Están preparados para aplicar para su niño que tiene 3 o 4 años?

**No translation currently for...** 
"Enrolling your 3 or 4 year-old child in .... Chicago Early Learning Hotline: 312-229-1690"
"Have you found the right program for your child?"

**Translation, that will require updating
La ciudad de Chicago y sus compañeros ofrece los programas preescolares en más de 600 centros en la ciudad. Luego, visite la aplicación en el internet para adquirir una cuenta y perfil familiar, crear una aplicación para su hijo(s), y seleccionar su centro(s) preferido(s). 


... more translations or recommendations to come for this page.

**Comment by theandrewbriggs** [comment]
For pages like these, I don't use Django's default translation mechanic because it's bad for huge blocks of text; instead, I'll add the translations when we have complete text in Spanish.

**Comment by smarziano** [comment]
@theandrewbriggs  Here is the Spanish with the English text filling any gaps. I requested some assistance for translation. Once this page is complete I want to review and see if we need to make any adjustments because it does have a lot of information.


# ¡Aplique hoy!
_Si usted está interesado(a) de un programa para niños recién nacidos hasta la edad de tres años, comuníquese con su proveedor directamente._

## ¿Están preparados para aplicar para su niño que tiene 3 o 4 años?

Enrolling your 3 or 4 year-old child in an early learning program is easy, and open year-round. Follow these easy steps to get your child enrolled in preschool:

Step 1) Click Here to create an online account for your family

Step 2) Fill in information on your family, such as employment and income, information on your child, and then apply for your top choice preschool site

Step 3) If your child qualifies, you will receive an email on June 1st with information on how to verify your child’s preschool seat

See you for the first day of school in September!

If you have questions, please call our Chicago Early Learning Hotline: 312-229-1690.

## **Have you found the right program for your child?**

La ciudad de Chicago y sus compañeros ofrece los programas preescolares en más de 600 centros en la ciudad. Luego, visite la aplicación en el internet para adquirir una cuenta y perfil familiar, crear una aplicación para su hijo(s), y seleccionar su centro(s) preferido(s).

## Need application support?

También puede llamar a la Línea de Aprendizaje Temprano de Chicago en (312) 229-1690 y someter una solicitud por teléfono o visite [un centro de recurso familiar](https://chicagoearlylearning.org/faq#lista-de-centro-de-recursos-familiares).

## How will you verify your child's placement?

If your child is offered a preliminary preschool placement, you will receive an email with verification instructions.

Follow these three steps to accept your child’s placement:

Step 1) Gather Documents

*see below for a list of document types that are generally accepted Proof of your child’s birth Examples: Birth certificate, passport or state issued medical card Proof of employment or other income for all family members listed on the application – Examples: W-2, two pay-stubs

Step 2) Bring and verify Documents

If your child has a seat at a Chicago Public School (CPS) site

**[insert table]**

If your child has a seat at a Community-based site bring your documents directly to the site which your child was assigned.

Bring your documents to the Community-Based Site at which your child was assigned

Step 3) Bring your child on the first day of school in September. Let the learning begin!

### Lista de documentos calificados para colocación
Un centro puede solicitar documentación adicional antes de completar la inscripción.

#### Prueba de edad del niño(a)

- Certificado original de nacimiento
- Pasaporte

#### Proof of address

- Licencia de conducir/tarjeta de identificación
- Cuenta de utilidades
- Contrato de arrendamiento/hipoteca
- Tarjeta de matriculación del votar

#### Prueba de ingresos

- Talonario reciente de pago
- Forma 1040/W-2
- SSI carta (Seguridad de Ingreso Suplementario)
- Carta de desempleo/talonario de cheque
- Carta compensación a trabajadores/talonario de cheque
- Carta de retiro o pensión/talonario de cheque
- TANF (El Programa de Asistencia Temporal Para Familias Necesitadas)

_Note: If your family is in a temporary living situation, you are not required to present the documents listed above for verification. All schools must immediately enroll students in temporary living situations even if the student lacks health, immunization or school records, proof of guardianship, proof of residency, or any other documentation normally required for school enrollment._

**Comment by smarziano** [comment]
@theandrewbriggs A lot of great work for this page. Here are some changes:

- [ ] Center the "Step 1, 2, 3..." text under "Are you ready to apply for preschool for your 3 or 4 year old?"
- [ ] Lower-case "here" in "Click Here" under "Are you ready to apply for preschool for your 3 or 4 year old?"
- [ ] Add telephone hyperlink to all instance of hotline phone number on page
- [ ] Do not add a screenshot, but remove this text "[Add screenshot of compare & apply tool that is on the homepage]" Reasoning is that this page has a LOT of information and I would like to reconsider with in the next milestone
- [ ] Change this "*see below for a list of document types that are generally accepted Proof of your child’s birth – Examples: Birth certificate, passport or state issued medical card Proof of employment or other income for all family members listed on the application – Examples: W-2, two pay-stubs" to "*see below for a list of document types that are accepted as proof of your child's birth and proof of employment or other income"
- [ ] Move this "If your child has a seat at a Community-based site bring your documents directly to the site which your child was assigned." to before "If your child has a seat at a Chicago Public School (CPS) site..."
- [ ] Delete "Bring your documents to the Community-Based Site at which your child was assigned" this is a duplicate sentence
- [ ] Change "If your child has a seat at a Chicago Public School (CPS) site" to "If your child has a seat at a **Chicago Public School (CPS) site** bring your documents to a Family Resource Center."

**Comment by smarziano** [comment]
@theandrewbriggs "If your child has a seat at a Chicago Public School (CPS) site bring your documents to a Family Resource Center." This sentence appears to be missing. It should go directly under "If your child has a seat at a Community-based site bring your..." and above the Family Resource Center chart.

**Comment by smarziano** [comment]
This has been updated. Closing issue!

We recently received Spanish translations. I will be creating a separate issue for that.
